sql/mob_spawn_mods.sql

```diff
@@ -605,16 +605,16 @@ INSERT INTO `mob_spawn_mods` VALUES (17236201,55,180,1); -- IDLE_DESPAWN: 180
 -- Boreal Hound
 INSERT INTO `mob_spawn_mods` VALUES (17236202,12,15,1);  -- DRAW_IN: 15
 INSERT INTO `mob_spawn_mods` VALUES (17236202,160,50,0); -- DMG: 50
-INSERT INTO `mob_spawn_mods` VALUES (17236202,169,10,0); -- MOVE: 10
+INSERT INTO `mob_spawn_mods` VALUES (17236202,75,10,0); -- MOVE_SPEED_STACKABLE: 10
```

Comment:
actually these should _exist_

mobs were never supposed to use what was there before anyway. please delete.

the ones in scripts will need nuked later as well - mobs should be having their speeds correctly set in their sql, or calling `setSpeed()` when they need to change on the fly or when sql isn't an option.

Reply:
Weshould really just move all the mods here to their mob scrits and be done with this table.
That's out of the scope of this PR though, of course.

Reply:
yeah but the speed ones don't belong there either. the rest tho, agreed death to this table.